### ISSUE TITLE
Cannot escape a hash immediately before a number

### ISSUE THREAD

**Issue by AJNeufeld** [mojombo/github-flavored-markdown#61]
Escaping the hash (#) does not work, when followed by digits.

I'd like to write about a transformer winding, specifically, winding number 1 or winding number 2.
Of course, writing `... transformer winding #1 is ...` results in `#1` being treated as an issue link to Issue #1, which (on this site) is "Github Flavored Markdown Examples", totally unrelated to power transformers. :wink:

However, escaping the hash does not seem to be possible: `\#1` still results in #1, the issue link.

Using HTML entities for the # or digits does not fool the Markdown parser into not seeing character sequence as an issue link:
- `&#35;1` results in the link &#35;1
- `&#35;&#49;` also results in &#35;&#49;

**Comment by chocolateboy** [comment]
A hacky workaround is to insert a zero width character between the hash and the number e.g.  `gulpjs\/gulp#&#8203;4.0` → gulpjs\/gulp#&#8203;4.0.

Of course, if someone copies 'n' pastes this, they will wind up with an unexpected [zero-width space](https://en.wikipedia.org/wiki/Zero-width_space) in their text...

**Comment by jefersonchaves** [comment]
I know this is pretty old and perhaps not a top priority, but was there any update?

**Comment by AJNeufeld** [comment]
On our private gitlab server, `Transformer \#1` no longer links to issue #1, so it seems to be fixed in some code bases and is no longer an issue for us personally, 👍 

... but still seems to be an issue here on GitHub!  😆

**Comment by chipbite** [comment]
2024 says hello, this is still an issue, AFAICS. `\#1` does not escape the hash - a reference to issue nr 1 is still created. 
So no way to write e g `#1` in plaintext in github markdown, AFAIK. 

Or any other tips? The zero-width character workaround is not reasonable for my usage, I will simply put it in backtick/code blocks

**Comment by zachpoblete** [comment]
@chipbite `<span>#</span>1` produces <span>#</span>1

[Source](https://stackoverflow.com/a/27394818/21138349)

**Comment by chipbite** [comment]
> @chipbite `<span>#</span>1` produces <span>#</span>1
> 
> [Source](https://stackoverflow.com/a/27394818/21138349)

Yeah, true, but I think that is just far too bloated. Backslash would be nice, as stated.
